addons/project/static/tests/project_subtask_kanban_list_tests.js

```diff
@@ -0,0 +1,81 @@
+/** @odoo-module */
+import {
+    click,
+    getFixture,
+} from '@web/../tests/helpers/utils';
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+import { start, startServer } from '../../../mail/static/tests/helpers/test_utils';
+import { addModelNamesToFetch } from '@bus/../tests/helpers/model_definitions_helpers';
+
+addModelNamesToFetch([
+    'project.project',
+    'project.task',
+]);
+
+let target;
+
+QUnit.module('Subtask Kanban List tests', {
+    beforeEach: async function () {
+        const pyEnv = await startServer();
+        const projectId = pyEnv['project.project'].create([
+            {name: "Project One", allow_subtasks: true}
+        ]);
+        const userId = pyEnv['res.users'].create([
+            {name: "User One", login: 'one', password: 'one'}
+        ]);
+        pyEnv['project.task'].create([
+            {name: 'task one', project_id: projectId, subtask_count: 2, child_ids: [2, 3], kanban_state: 'normal', user_ids: [userId]},
+            {name: 'task two', subtask_count: 0, child_ids: [], kanban_state: 'done'},
+            {name: 'task three', subtask_count: 0, child_ids: [], kanban_state: 'blocked'},
+            {name: 'task four', project_id: projectId, subtask_count: 0, child_ids: [], kanban_state: 'normal'},
+        ]);
+        this.views = {
+            "project.task,false,kanban": 
+                `<kanban js_class="project_task_kanban">
+                    <field name="subtask_count"/>
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div>
+                                <field name="name" widget="name_with_subtask_count"/>
+                                <field name="user_ids" invisible="1"/>
+                                <a t-if="record.subtask_count.raw_value" class="subtask_list_button" title="See Subtasks" style="width: 29px;"/>
+                            </div>
+                            <div t-if="!state.folded">
+                                <widget name="subtask_kanban_list"/>
+                            </div>
+                        </t>
+                    </templates>
+                </kanban>`,
+        };
+        target = getFixture();
+        setupViewRegistries();
+    }
+}, function () {
+    QUnit.test("Check whether subtask list functionality works as intended", async function (assert) {
+        assert.expect(8);
+
+        const views = this.views;
+        const { openView } = await start({
+                                    serverData: { views }
+                                });
```

Comment:
Since it is not really long we could write it in one line. 🙂 😇 
```suggestion
        const { openView } = await start({ serverData: { views } });
```